lib/src/syn/v1/literal/algorithm.rs

```diff
@@ -17,5 +17,7 @@ pub fn algorithm(i: &str) -> IResult<&str, Algorithm> {
 		value(Algorithm::Rs256, tag("RS256")),
 		value(Algorithm::Rs384, tag("RS384")),
 		value(Algorithm::Rs512, tag("RS512")),
+		#[cfg(feature = "http")]
+		value(Algorithm::Jwks, tag("JWKS")), // Not an algorithm.
```

Comment:
I think it is probably better to have the parser always be able to parse this regardless of whether the feature is actually enabled. If we disable this path when `http` is not enabled the parser will just throw a syntax error which might be more confusing then an error message explaining that this feature is not available in the build. This is the approach we have been mostly using for optional features.

Reply:
That's a great point, I will include it in any case. Thanks! 👍

Reply:
I have handled this in [239ef33](https://github.com/surrealdb/surrealdb/pull/3198/commits/239ef338bb7103d98b3720670a653b7c2d523c61). Is that what you had in mind?

Reply:
I had in mind to just always parse the `JWKS` keyword to `Alrgorithm::Jwks` and handle any missing features when the query is actually executed. 

Since the rust sdk parses the query before sending it to the server this way of doing that disallows a client which isn't compiled with the `http` feature to run a query with the `JWKS` keyword when it should be able to, I think.

Reply:
Yep that is what I had in mind!